api_deploy.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 
-# clones the api from github using HTTPS, install all deps
+# This script clones the expressapi repo from github using HTTPS, and installs dependencies
+
+sudo apt install net-tools
```

Comment:
What's this for?

Reply:
This guide may help you set it up better:

https://www.digitalocean.com/community/tutorials/how-to-install-nginx-on-ubuntu-20-04